### PR TITLE
fix: Fixes EvilMage/Lord spawning naked, without hair

### PR DESCRIPTION
**Pull request by bohica89**

### CODE REVIEW

File: Projects/UOContent/Mobiles/Monsters/Humanoid/Magic/EvilMage.cs [hunk 1]
```diff
@@ -39,8 +40,17 @@ public EvilMage() : base(AIType.AI_Mage)
...
+            if (Core.UOR)
+            {
+                PackItem(new Robe(Utility.RandomNeutralHue()));
+                PackItem(new Sandals());
+            }
+            else
+            {
+                EquipItem(new Robe(Utility.RandomNeutralHue()));
+                EquipItem(new Sandals());
+            }
```
Comment (kamronbatman) [inline Projects/UOContent/Mobiles/Monsters/Humanoid/Magic/EvilMage.cs:53]: Let's just equip it.

```suggestion
            EquipItem(new Robe(Utility.RandomNeutralHue()));
            EquipItem(new Sandals());
```

File: Projects/UOContent/Mobiles/Monsters/Humanoid/Magic/EvilMage.cs [hunk 2]
```diff
@@ -12,6 +12,7 @@ public EvilMage() : base(AIType.AI_Mage)
             Name = NameList.RandomName("evil mage");
             Title = "the evil mage";
             Body = Core.UOR ? 124 : 0x190;
+            Hue = Core.UOR ? 0 : Race.Human.RandomSkinHue();
```
Comment (kamronbatman) [inline Projects/UOContent/Mobiles/Monsters/Humanoid/Magic/EvilMage.cs:15]:
```suggestion
            if (Core.UO)
            {
                Hue = Race.Human.RandomSkinHue();
            }
```

Comment (bohica89) [reply Projects/UOContent/Mobiles/Monsters/Humanoid/Magic/EvilMage.cs:15]: Actually, I think EvilMages and EvilMageLords only had the pale models (0x190) and no hue switching.

Comment (kamronbatman) [reply Projects/UOContent/Mobiles/Monsters/Humanoid/Magic/EvilMage.cs:15]: Then let's get rid of the hue change?

**Pull request merged**